### ISSUE TITLE
deploy.sh overwrites VmSize settings for master(s) and node(s)

### ISSUE THREAD

**Issue by alexbez** [mglantz/ocp36-azure-simple#12]
Line 58 and down:

"masterVmSize": {
                        "value": "Standard_DS4_v2"
                },
                "nodeVmSize": {
                        "value": "Standard_DS3_v2"
                },

So, no matter what you put in your 'azuredeploy.parameters.json', types/sizes of VMs would be overwritten by these hardcoded values. This is why when I tried to deploy 1 master, 1 infra, 1 node cluster using smaller instances, it still tried to allocate DS4 and DS3 v2 VMs.

**Comment by mglantz** [comment]
Yeah, this needs to be fixed, but, it needs to be in sync with minimum allowed sizes of OCP. The masterVMsize minimum spec is Standard_DS4_v4 and node minimum spec is DS3_v3.
